autowebcompat/utils.py

```diff
@@ -190,6 +190,19 @@ def read_labels(file_name='labels.csv'):
     return labels
 
 
+def Classify(label, classification_type='Y vs D + N'):
```

Comment:
**Classify** is not a great name for the function, use something like **to_categorical_label**.